cocos/ui/components/widget-component.ts

```diff
@@ -320,6 +360,10 @@ export class WidgetComponent extends Component {
     }
 
     /**
+     * @en
+     * Whether the stretched horizontally, when enable the left and right alignment will be stretched horizontally,
```

Comment:
Whether the stretched horizontally, when enable the left and right alignment will be stretched horizontally,
-->
Whether to stretch horizontally, when enabled the left and right alignment will be stretched horizontally,